# request
geometry_msgs/PoseStamped[] starts
geometry_msgs/PoseStamped[] goals
---
# response
nav_msgs/Path[] paths